iis-wf/iis-wf-ingest-pmc/src/main/java/eu/dnetlib/iis/wf/ingest/pmc/metadata/RefListXmlHandler.java

```diff
@@ -70,7 +70,8 @@ public void startElement(String uri, String localName, String qName, Attributes
             this.currentValue.setLength(0);
         } else if (isWithinElement(qName, ELEM_PUB_ID, parents, ELEM_CITATION) ||
                 isWithinElement(qName, ELEM_PUB_ID, parents, ELEM_ELEMENT_CITATION) ||
-                isWithinElement(qName, ELEM_PUB_ID, parents, ELEM_MIXED_CITATION)) {
+                isWithinElement(qName, ELEM_PUB_ID, parents, ELEM_MIXED_CITATION) ||
```

Comment:
I can see several added lines including `ELEM_MIXED_CITATION` which was supposed to be already supported. Does it mean we didn't fully support it yet?

If so, I guess we should indicate it in the ticket description and the commit message so we won't make any `nlm-citation` unrelated change in this commit without making it more visible (otherwise it might be difficult to track changes in future and make debugging more difficult).

Reply:
`mixed-citation` elements are already supported, this PR only adds support for `nlm-citation` elements. I think github's diff misleaded you, the lines with `ELEM_MIXED_CITATION` are marked as changed because of changes needed to add support for `nlm-citation` elements.